feed-integration-webapp/src/main/java/org/bahmni/module/feedintegration/atomfeed/mappers/OpenMRSPatientMapper.java

```diff
@@ -0,0 +1,38 @@
+package org.bahmni.module.feedintegration.atomfeed.mappers;
+
+import org.bahmni.module.feedintegration.atomfeed.contract.patient.OpenMRSPatient;
+import org.bahmni.module.feedintegration.atomfeed.contract.patient.OpenMRSPatientFullRepresentation;
+import org.bahmni.webclients.ObjectMapperRepository;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+
+public class OpenMRSPatientMapper {
+    private ObjectMapper objectMapper;
+    private SimpleDateFormat dateOfBirthFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
+
+    public OpenMRSPatientMapper() {
+        this.objectMapper = ObjectMapperRepository.objectMapper;
+    }
+
+    public OpenMRSPatient map(String patientJSON) throws IOException, ParseException {
+        OpenMRSPatient patient = new OpenMRSPatient();
+        JsonNode jsonNode = objectMapper.readTree(patientJSON);
+
+        patient.setPatientId(jsonNode.path("identifiers").get(0).path("identifier").asText());
```

Comment:
The mapper here seems to be correct. Uses minimal information. Maybe its just the OpenMRS Person/Patient object then that needs to drop the extra getter/setters from its definition.